CONTRIBUTING.md

```diff
@@ -16,8 +16,14 @@ the preferred channel for submitting [pull requests](#pull-requests) and
 [bug reports](#bugs), but please respect the following
 restrictions:
 
-* Please **do not** use the issue tracker for personal support requests (use
-  [mailing list](http://www.pcl-users.org/)).
+* Please **do not** use the issue tracker for personal support requests. Please choose one of the following alternatives instead:
+  * Q&A at [Stack Overflow](https://stackoverflow.com/questions/tagged/point-cloud-library):
+  Do remember to tag your questions with the tag `point-cloud-library`
+  * Live chat on the [Gitter chanel](http://www.pcl-users.org/)
```

Comment:
Please fix the link here. Also, chan**n**el.

Reply:
Oops. On it